tests/cli/dummy_features/__snapshots__/clean_snapshots.feature.snap

```diff
@@ -2,6 +2,3 @@
 
 exports[`Should echo salut 1.1`] = `"salut
 "`;
-
-exports[`Should get deleted 1.1`] = `"salut
```

Comment:
Please revert this

Reply:
I think this happens when I use command `yarn lint-fix`, I'll check that.

Reply:
Actually that should be `yarn test-cli`.

It was added in https://github.com/ekino/veggies/pull/69, including some functional testing on snapshots operations, like deletion.

They are nice overhaul, but we might need to improve them to avoid commiting the dummy snapshots.
In the meantime, we just need to check if they were modified like I just did. 😄 

By the way, the test shouldn't pass in this state anyway. Since you're a first time contributor, I had to manually validate your workflow, we'll see if the tests are working correctly. 😬 

Welcome to the team by the way ! 🎉

Reply:
Well, looks like we have an issue with `yarn run check-readme`, that's anticlimactic. 😓 

I'll check what's up.

Reply:
That's just because you didn't run `yarn readme` @tienduy-nguyen, no real issue there.

README.md is generated from a template and our Gherkin definitions. See https://github.com/ekino/veggies/blob/master/CONTRIBUTING.md#updating-readmemd for more info.

Reply:
Yes, I reverted it already.